code/modules/projectiles/projectile/bullets.dm

```diff
@@ -203,13 +203,8 @@
 //This one is for future syringe guns update
 /obj/item/projectile/bullet/dart/syringe
 	name = "syringe"
-	icon = 'icons/obj/chemical.dmi'
 	icon_state = "syringeproj"
 
-//Piercing Syringe
-/obj/item/projectile/bullet/dart/syringe/piercing
-	piercing = 1

```

Comment:
removed to simplify code, we now use a regular syringe dart but with piercing set to 1 (we check if the syringe item used is a piercer or not (new piercing_proj var)